code_schemes/s01e01.json

```diff
@@ -33,6 +33,120 @@
         "VisibleInCoda": true,
         "Shortcut": "s"
       },
+      {
+        "CodeID": "code-S-643dc650",
+        "CodeType": "Meta",
+        "MetaCode": "answer",
+        "DisplayText": "meta: answer",
+        "NumericValue": -1000140,
+        "StringValue": "answer",
+        "VisibleInCoda": true,
+        "Shortcut": "a"
+      },
+      {
+        "DisplayText": "yes, no reason",
+        "VisibleInCoda": true,
+        "NumericValue": 1,
+        "CodeID": "code-d6a975c6",
+        "CodeType": "Normal",
+        "StringValue": "yes_no_reason"
```

Comment:
This might also be confusing, but can't think of anything better so ok with it RDA are happy.